common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -173,7 +173,7 @@ class BulkOperationsMixin(object):
     mongo_connection.
     """
     def __init__(self, *args, **kwargs):
-        super(BulkOperationsMixin, self).__init__(*args, **kwargs)
+        super(BulkOperationsMixin, self).__init__()
```

Comment:
I think I likely allowed `*args` and `**kwargs` through because it might be mixed in with other base classes that do need those arguments. In particular, consider something like:

```
class ModulestoreBase(object):
    def __init__(self, some_arg):
        ...

class ModulestoreA(BulkOperationsMixin, ModulestoreBase):
    ...

class ModulestoreB(ModulestoreBase, BulkOperationsMixin):
    ...
```

If I have my mental simulation of the MRO correct, for `ModulestoreA`, it's `ModulestoreA -> BulkOperationsMixin -> ModulestoreBase -> object`, and for `ModulestoreB`, it's `ModulestoreB -> ModulestoreBase -> BulkOperationsMixin -> object`. For `ModulestoreA`, the version of `BulkOperationsMixin` as written is correct, so that arguments get passed to `ModulestoreBase`. For `ModulestoreB`, the version of `BulkOperationsMixin` as presented in this patch is correct, so that arguments aren't passed on to `object`.

I am of the opinion that the mixins should come before the baseclass (which makes the code as written the correct choice).